src/FsAutoComplete.BackgroundServices/Program.fs

```diff
@@ -59,6 +59,9 @@ module Helpers =
         | FSharpDiagnosticSeverity.Hidden -> None
         | FSharpDiagnosticSeverity.Info -> Some DiagnosticSeverity.Information
 
+    let urlForCompilerCode (number: int) =
```

Comment:
duplicated logic, would be nice to reduce the amount of that between these two apps.